src/components/Video/Video.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Video = (props) => {
+  const { src, classes } = props;
+  return (
+    <video controls src={src} className={classes}>
+      <track kind="captions" />
+    </video>
+  );
+};
+Video.propTypes = {
+  src: PropTypes.string.isRequired,
+  classes: PropTypes.string.isRequired,
+};
+
+export default Video;
```

Comment:
@mcaleb808 @Nziranziza @filschristian ????

Reply:
@kimotho-njoki Could you help us understand what you mean?